src/scene/constants.js

```diff
@@ -868,3 +868,24 @@ export const ORIENTATION_HORIZONTAL = 0;
  * @type {number}
  */
 export const ORIENTATION_VERTICAL = 1;
+
+/**
+ * A sky texture is rendered using an infinite projection.
+ *
+ * @type {string}
+ */
+export const SKYMESH_INFINITE = 'infinite';
```

Comment:
Since these are set on a property called `type`, would `SKYTYPE_*` be better naming?